pom.xml

```diff
@@ -28,7 +28,7 @@
 
   <properties>
     <jenkins.version>1.580.3</jenkins.version>
-    <java.level>6</java.level>
+    <java.level>7</java.level>
```

Comment:
Also needs bumping the core dependency then. I would propose to pick 1.625.x as a first version with Java 7